core/src/test/scala/unit/kafka/security/authorizer/AuthorizerTest.scala

```diff
@@ -786,291 +641,17 @@ class AuthorizerTest extends QuorumTestHarness with BaseAuthorizerTest {
-    val prefixedValue = "prefixedValue"
-    val prefix = "authorizer."
-    val configs = Map("zookeeper.connect" -> "somewhere", // required, otherwise we would omit it
-      ZkConfigs.ZK_SSL_CLIENT_ENABLE_CONFIG -> "false",
-      ZkConfigs.ZK_CLIENT_CNXN_SOCKET_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_KEY_STORE_LOCATION_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_KEY_STORE_PASSWORD_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_KEY_STORE_TYPE_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_TRUST_STORE_LOCATION_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_TRUST_STORE_PASSWORD_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_TRUST_STORE_TYPE_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_PROTOCOL_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_ENABLED_PROTOCOLS_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_CIPHER_SUITES_CONFIG -> kafkaValue,
-      ZkConfigs.ZK_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG -> "HTTPS",
-      ZkConfigs.ZK_SSL_CRL_ENABLE_CONFIG -> "false",
-      ZkConfigs.ZK_SSL_OCSP_ENABLE_CONFIG -> "false",
-      prefix + ZkConfigs.ZK_SSL_CLIENT_ENABLE_CONFIG -> "true",
-      prefix + ZkConfigs.ZK_CLIENT_CNXN_SOCKET_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_KEY_STORE_LOCATION_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_KEY_STORE_PASSWORD_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_KEY_STORE_TYPE_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_TRUST_STORE_LOCATION_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_TRUST_STORE_PASSWORD_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_TRUST_STORE_TYPE_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_PROTOCOL_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_ENABLED_PROTOCOLS_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_CIPHER_SUITES_CONFIG -> prefixedValue,
-      prefix + ZkConfigs.ZK_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG -> "",
-      prefix + ZkConfigs.ZK_SSL_CRL_ENABLE_CONFIG -> "true",
-      prefix + ZkConfigs.ZK_SSL_OCSP_ENABLE_CONFIG -> "true")
-    configs.foreach{case (key, value) => props.put(key, value.toString) }
-
-    val zkClientConfig = AclAuthorizer.zkClientConfigFromKafkaConfigAndMap(
-      KafkaConfig.fromProps(props), mutable.Map(configs.toSeq: _*))
-    // confirm we get all the values we expect
-    ZkConfigs.ZK_SSL_CONFIG_TO_SYSTEM_PROPERTY_MAP.asScala.keys.foreach(prop => prop match {
-      case ZkConfigs.ZK_SSL_CLIENT_ENABLE_CONFIG | ZkConfigs.ZK_SSL_CRL_ENABLE_CONFIG | ZkConfigs.ZK_SSL_OCSP_ENABLE_CONFIG =>
-        assertEquals("true", KafkaConfig.zooKeeperClientProperty(zkClientConfig, prop).getOrElse("<None>"))
-      case ZkConfigs.ZK_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG =>
-        assertEquals("false", KafkaConfig.zooKeeperClientProperty(zkClientConfig, prop).getOrElse("<None>"))
-      case _ => assertEquals(prefixedValue, KafkaConfig.zooKeeperClientProperty(zkClientConfig, prop).getOrElse("<None>"))
-    })
-  }
-
-  @Test
```

Comment:
Are these tests not relevant in KRaft mode?

Reply:
This case overrides `AclAuthorizer#processAclChangeNotification`. IIRC, we don't have similar mechanism in `StandardAuthorizer`.